conformance-suites/2.0.0/conformance2/renderbuffers/00_test_list.txt

```diff
@@ -1,5 +1,5 @@
 framebuffer-object-attachment.html
-framebuffer-test.html
```

Comment:
Leave the 2.0.0 snapshot alone, actually.

Reply:
Done.